chips/earlgrey/src/chip.rs

```diff
@@ -399,7 +399,7 @@ pub extern "C" fn _start_trap_vectored() {
 }
 
 #[cfg(all(target_arch = "riscv32", target_os = "none"))]
-#[link_section = ".riscv.trap_vectored"]
+#[link_section = ".riscv.open_titan_trap_vectored"]
```

Comment:
why break from the convention used elsewhere for just OT?

Reply:
I thought I did this to allow signing the image. I can't find any evidence of this though and we aren't signing it so I'll remove this